cpp/inc/bond/stream/output_counter.h

```diff
@@ -66,26 +77,40 @@ class OutputCounter
     {
         _count += sizeof(T);
     }
-    
+
     void Write(const void*, uint32_t size)
     {    
         _count += size;
     }
-    
+
     void Write(const blob& buffer)
     {
         _count += buffer.size();
     }
-    
+
+    void Write(const Buffer& buffer)
+    {
+        _count += buffer.size();
+    }
+
     template<typename T>
     void WriteVariableUnsigned(T value)
     {
         VariableUnsigned<T, 1>::Write(_count, value >> 7);
     }
 
+    Buffer GetBuffer() const
+    {
+        return{ GetCount() };
```

Comment:
Minor: `return { GetCount() } ;`
Missing space between keyword and initializer list.